app/utils/graphql.ts

```diff
@@ -67,33 +97,31 @@ export const useQuery = <T extends Record<string, unknown>>({
   variables,
   options,
   typename,
-}: {
-  query: RequestDocument | null;
-  variables?: Variables;
-  options?: SWRConfiguration;
-  typename: string;
-}): UseQueryResponse<T> => {
+  hasAggregates,
+}: UseQueryProps): UseQueryResponse<T> => {
   const { user } = useAuth0();
   const token = useToken();
 
   const fetchWithAuth = async ([query, variables]: [
     RequestDocument,
     Variables
-  ]): Promise<{ [K in typeof typename]: T[] }> => {
+  ]): Promise<HasuraGraphQLResponse<T, typeof typename>> => {
```

Comment:
Before this change, we were telling SWR that our query always returned an object with a single property that matches the given `typename` and array of whatever Type (`T`), e.g.

```javascript
{ crashes : []}
```

Now the type looks like:

```javascript
{ crashes : [], crashes_aggregate: {} }
```